src/docs/product/relay/modes.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Relay Modes
+sidebar_order: 1
+---
+
+When configuring a Relay server, the most important thing it is to understand
+the major modes in which it can operate.
+
+As seen in the introduction, the configuration file contains a `relay.mode`
+field. Currently, there are three modes that can be specified: `managed`,
+`static`, `proxy`.
+
+Event processing in Sentry can be configured using a two level hierachical
+configuration. At the top level, there are organization wide settings. An
```

Comment:
This is a bit confusing, so here's a suggested revision:

Event processing in Sentry is configured according to settings defined for the organization and project. Some settings, such as privacy controls, are set at the organization level and inherited by all projects in that organization; other settings are specified per project. For Relay, events are processed according to the project settings to which the event belongs. The mode controls the way Relay obtains these project settings.